data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -205,24 +205,24 @@ def _workforce_factor(self) -> bool:
         # (necessary to screen out university block groups)
         workforce_criteria = (
             (
-                self.df[field_names.UNEMPLOYMENT_FIELD]
+                self.df[field_names.UNEMPLOYMENT_PERCENTILE_FIELD]
                 > self.ENVIRONMENTAL_BURDEN_THRESHOLD
             )
             | (
-                self.df[field_names.MEDIAN_INCOME_PERCENT_AMI_FIELD]
+                self.df[field_names.MEDIAN_INCOME_PERCENT_AMI_PERCENTILE_FIELD]
                 # Note: a high median income as a % of AMI is good, so take 1 minus the threshold to invert it.
                 # and then look for median income lower than that (not greater than).
                 < 1 - self.ENVIRONMENTAL_BURDEN_THRESHOLD
             )
             | (
-                self.df[field_names.POVERTY_LESS_THAN_100_FPL_FIELD]
+                self.df[field_names.POVERTY_LESS_THAN_100_FPL_PERCENTILE_FIELD]
                 > self.ENVIRONMENTAL_BURDEN_THRESHOLD
             )
             | (
-                self.df[field_names.LINGUISTIC_ISO_FIELD]
+                self.df[field_names.LINGUISTIC_ISO_PERCENTILE_FIELD]
                 > self.ENVIRONMENTAL_BURDEN_THRESHOLD
             )
         )
         return (
-            self.df[field_names.HIGH_SCHOOL_ED_FIELD] > 0.05
+            self.df[field_names.HIGH_SCHOOL_ED_PERCENTILE_FIELD] > 0.05
```

Comment:
Ah! This is the rare one where it's actually supposed to stay as `field_names.HIGH_SCHOOL_ED_FIELD`.